mirror_pso_containers.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Script to mirror PSO required containers into a local Registry
+
+# Ensure you have logged in to your local registry using 'docker login' before using this script
+
+REGISTRY_URL=<your local registry address, including port>
+
+LIST=(
+quay.io/k8scsi/csi-provisioner:v1.6.0
+quay.io/k8scsi/csi-snapshotter:v2.1.1
+quay.io/k8scsi/csi-attacher:v2.2.0
+quay.io/k8scsi/csi-resizer:v0.5.0
+quay.io/k8scsi/livenessprobe:v2.0.0
+quay.io/k8scsi/csi-node-driver-registrar:v1.3.0
+purestorage/cockroach-operator:v1.0.2
+purestorage/dbdeployer:v1.0.2
+purestorage/psctl:v1.0.0
+purestorage/k8s:v6.0.2
+cockroachdb/cockroach:v19.2.3
+)
+
+newImageRepo="${REGISTRY_URL}\/library"
```

Comment:
does dtr repo path alway have `/library`?

Reply:
Harbor as a local registry certainly uses this as a default. I haven't checked other repositories

Reply:
Nice. Users can customized the script if they use other type of repos. 
Like the idea to automate this process.